### DISCUSSION TITLE
Cascade Traffic or Change ports

### DISCUSSION THREAD

**Discussion by JellyVPN** [strongswan/strongswan#1368] [Q&A]
Hi Everyone, I appreciate any help
there is any possibility to have two strongSwan servers but users connect to Server A and get traffic from server B
1. In Iran, and China, high censorship, they can't access outside of the country, so people living in Iran can't connect to the server I have in Europe, I should have a server in Iran in a VPS then all data from that server forward to Europe server
2. users after connecting to the A (Iran) server get B (Europe) server
3. how can I use the radius server?
==============
or when the port used normally by IKEv2 is blocked in Iran is it possible to change ports and use it, maybe changing ports works well
I appreciate any help

**Comment by Thermi** [comment]
There is. Just configure it that way. Remember the networks in the config are used verbatim.

RE Radius server: configure eap-radius and use it as eap authentication method. See https://docs.strongswan.org/docs/5.9/plugins/eap-radius.html
